lib/index.ts

```diff
@@ -156,8 +156,58 @@ interface ServerOptions extends EngineAttachOptions {
   connectTimeout: number;
 }
 
-export class Server extends EventEmitter {
-  public readonly sockets: Namespace;
+/**
```

Comment:
How about moving those declarations in their own file? (completely cosmetic, it's your call :angel: )

Reply:
I'm all for it! With the addition of `StrictEventEmitter` I think it made a lot of sense to move it all to a separate file.